redhat/frr.spec.in

```diff
@@ -674,11 +699,16 @@ fi
 
 %files pythontools
 %{_sbindir}/generate_support_bundle.py
+%{_sbindir}/frr-reload.py
+%if 0%{?rhel} > 7 || 0%{?fedora} > 29
+%{_sbindir}/__pycache__/*
```

Comment:
I'm just curious: why is this needed here?

Reply:
RPMs are supposed to include the pyc/pyo files per policy.
If I use pathfix, they end up a cpython files in `__pycache__` directory.
Older versions create pyc/pyo directly in the same directory as the py file

See https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_packages_using_cython

Reply:
thanks explaining this